app/templates/auth/change-password.html

```diff
@@ -29,36 +29,48 @@ <h1 class="govuk-heading-xl">Change your password</h1>
       <div class="govuk-grid-column-two-thirds">
         {{ form.hidden_tag() }}
 
-        {% with question = form.old_password.label,
-                name = "old_password",
-                error = errors.get('old_password', {}).get('message'),
-                value = form.old_password.data,
-                type = "password",
-                autocomplete = "off"
-        %}
-          {% include "toolkit/forms/textbox.html" %}
-        {% endwith %}
+        {{ govukInput({
+          "label": {
+            "text": form.old_password.label.text,
+          },
+          "errorMessage": errors.get('old_password', {}).get('errorMessage'),
```

Comment:
Can we now simplify these to: 

`"errorMessage": errors.old_password.errorMessage,`

We have that pattern as an example in `dmutils.forms.helpers` - if we still need the old `get('name', {})` then we should change the example there...

Reply:
👍 I'll simplify these

Reply:
Done